lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -297,9 +297,8 @@ def _on_scrape_targets_changed(self, event):
 import json
```

Comment:
Kindly note, I think this PR should be associated with an update to the "Code Overview" section of the Contributing.md docs. There is already a pending Github issue for such and update from your previous PR. I think the best way to keep docs updated is to include them in the PR that updates the code.